Buildings/Applications/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -0,0 +1,152 @@
+    "Adding up integer signals"
+    annotation (Placement(transformation(extent={{70,-60},{90,-40}})));
+  Buildings.Controls.OBC.CDL.Conversions.IntegerToReal intToRea
+    "Integer to real"
+    annotation (Placement(transformation(extent={{40,-10},{60,10}})));
+
+equation
+  connect(off.outPort[1], offToOne.inPort) annotation (Line(points={{-50,59.5},{
+          -50,44}},               color={0,0,0}));
+  connect(oneToOff.outPort, off.inPort[1]) annotation (Line(points={{-20,41.5},{
+          -20,88},{-50,88},{-50,81}},     color={0,0,0}));
+  connect(oneToTwo.outPort, twoOn.inPort[1]) annotation (Line(points={{-50,-41.5},{-50,-59}}, color={0,0,0}));
+  connect(twoOn.outPort[1], twoToOne.inPort) annotation (Line(points={{-50,-80.5},
+          {-50,-88},{-2.22045e-16,-88},{-2.22045e-16,-44}},
+                                          color={0,0,0}));
+  connect(twoToOne.outPort, oneOn.inPort[2]) annotation (Line(points={{0,-38.5},
+          {0,16},{-49.5,16},{-49.5,11}},   color={0,0,0}));
+  connect(offToOne.outPort, oneOn.inPort[1]) annotation (Line(points={{-50,38.5},
+          {-50,24},{-50,11},{-50.5,11}}, color={0,0,0}));
+  connect(oneOn.outPort[2], oneToOff.inPort) annotation (Line(points={{-49.75,-10.5},
+          {-49.75,-18},{-20,-18},{-20,36}}, color={0,0,0}));
+  connect(oneOn.outPort[1], oneToTwo.inPort) annotation (Line(points={{-50.25,-10.5},
+          {-50.25,-18},{-50,-18},{-50,-36}}, color={0,0,0}));
+  connect(combiTable1Ds.y, y) annotation (Line(points={{91,0},{110,0}}, color={0,0,127}));
+  connect(combiTable1Ds.u, intToRea.y) annotation (Line(points={{68,0},{62,0}}, color={0,0,127}));
+  connect(addInt.u2, booToInt1.y) annotation (Line(points={{68,-56},{60,-56},{
+          60,-70},{52,-70}},
+                          color={255,127,0}));
+  connect(oneOn.active, booToInt.u) annotation (Line(points={{-39,0},{20,0},{20,
+          -30},{28,-30}}, color={255,0,255}));
+  connect(twoOn.active, booToInt1.u) annotation (Line(points={{-39,-70},{28,-70}}, color={255,0,255}));
+  connect(booToInt.y, addInt.u1) annotation (Line(points={{52,-30},{60,-30},{60,
+          -44},{68,-44}}, color={255,127,0}));
+  connect(addInt.y, intToRea.u) annotation (Line(points={{92,-50},{94,-50},{94,-14},
+          {34,-14},{34,0},{38,0}}, color={255,127,0}));
+  annotation (Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,
+            -100},{100,100}})),           Icon(coordinateSystem(
+          preserveAspectRatio=false, extent={{-100,-100},{100,100}}), graphics={
+                                        Text(
+        extent={{-150,150},{150,110}},
+        textString="%name",
+        lineColor={0,0,255})}),
+    Documentation(revisions="<html>
+<ul>
+<li>
+August 6, 2020 by Jing Wang:<br/>
+First implementation.
+</li>
+</ul>
+</html>", info="<html>
+<p>This model implements staging control logic of two chillers according to the measured total cooling load. The control logic is as follows:</p>
```

Comment:
Better clarify if the controller is applicable to chiller plant with (or without) waterside economizer.